api/pkg/common/common.go

```diff
@@ -30,7 +30,7 @@ const (
 	MaxPaginationLimit      = 1000
 	DefaultPaginationLimit  = MaxPaginationLimit
 	DefaultPaginationOffset = 0
-	MaxObjectSize           = 5 * 1024 * 1024 // 5GB
+	MaxObjectSize           = 5 * 1024 * 1024 * 1024 // 5GB
```

Comment:
Just confirming..1024 is the standard right? no 1000. please confirm GB or GiB

Reply:
@kumarashit ,  Thanks for your reviewingm,  I created a 5*1024*1024*1024 size file and put it to aws,  and aws response with ok, so it should be 1024